### DISCUSSION TITLE
Github Externship India

### DISCUSSION THREAD

**Discussion by dhivyasreedhar** [github/india#623] [Students]
My dashboard shows 504 error.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any `Official GitHub Externship Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏
